src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+// eslint-disable-next-line import/no-named-as-default
+import PeopleTable from '../PeopleTable/PeopleTable';
+import Person from '../../types/Person';
+
+const PeoplePage:React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [failedPeopleLoading, setFailedPeopleLoading] = useState(false);
+
+  // eslint-disable-next-line max-len
+  const BASE_URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
+
+  function getPeople<T>(): Promise<T> {
```

Comment:
also, it's `getPeople` method, not anything else, so it MUST return `Person[]` type, not anything else